### PR TITLE
Claim Status Request

### PR DESCRIPTION
**Pull request by teryhill**
Claim Status Request #304 . This is currently being tested by a clearing house.

### CODE REVIEW

File: library/edi_libre.inc [hunk 1]
```diff
@@ -0,0 +1,538 @@
+<?php
```
Comment (tmccormi) [inline library/edi_libre.inc]: Should this file be name edi_libre.inc?   Seems like it should be named in a way that references it purpose.

edi_276.inc.php perhaps.

**Comment by teryhill** [comment]
Changes made per your request. Tested on my local system (Using left nav)

**Comment by aethelwulffe** [comment]
What the heck is the librebot, and uh.....?????

**Comment by robbyoconnor** [comment]
Oh my god what's going on here

**Comment by teryhill** [comment]
The bot is rouge 😡

**Comment by aethelwulffe** [comment]
Le bot rouge?  It is a red robot?  Oh...rogue!!!

**Comment by aethelwulffe** [comment]
Oh _your_ God...they are finally taking over!

**Comment by teryhill** [comment]
No red. I got my laser and am hunting fer it.

**Comment by teryhill** [comment]
Danger Will Robinson

**Comment by aethelwulffe** [comment]
The computer is your friend.  Trust the computer, and keep your laser handy!

**Comment by robbyoconnor** [comment]
I disabled remote pushes...so @librebot  you better behave or I'll put you out of your misery :hocho:

**Comment by aethelwulffe** [comment]
Give 'em the clamps Robbie!

**Comment by teryhill** [comment]
Here mister bott nice mister bott 🔌.

**Comment by robbyoconnor** [comment]
@librebot you behave or else :knife: :bomb:

**Pull request merged**